config/filter.d/filter-ip-blacklist.conf

```diff
@@ -0,0 +1,43 @@
+[Definition]
+ 
+# Option:  failregex
+# Notes :  Detection of blocked ip addresses.
+# Values:  TEXT
+#
+ 
+failregex = ^(?P<ip4>\d{1,3}(?:\.\d{1,3}){3}/\d{1,2})\s+
```

Comment:
```suggestion
failregex = ^(?P<ip4>\d{1,3}(?:\.\d{1,3}){3}/\d{1,2})\s+\[(?P<timestamp>[^\]]+)\].*$
```

Reply:
1. For which reason this may be needed? 
    1.a. Normally it shall be matched by `datepattern`
    1.b. The part of messages matched by `datepattern` is cut out from message before `failregex` gets applied (see the note in [the manpage](https://manpages.debian.org/stable/fail2ban/jail.conf.5.en.html#failregex~2))
    1.c. The timestamp shall be better at 0 position in log if possible.
2. `.*$` is not an anchor and shall be removed or replaced with some non catch-all (like `\S*$`).